ocs_ci/deployment/deployment.py

```diff
@@ -207,13 +207,46 @@ class Deployment(object):
     """
 
     def __init__(self):
-        self.platform = config.ENV_DATA["platform"]
-        self.ocp_deployment_type = config.ENV_DATA["deployment_type"]
-        self.cluster_path = config.ENV_DATA["cluster_path"]
-        self.namespace = config.ENV_DATA["cluster_namespace"]
         self.sts_role_arn = None
-        self.storage_class = storage_class.get_storageclass()
-        self.custom_storage_class_path = None
+        storage_class.set_custom_storage_class_path()
```

Comment:
Does this need to be called from the deployment class or should this be handled fully within the storage_class module?

Reply:
I am calling it here to make sure when creating deployer object for each cluster config in cluster fixture - that we have properly set right away right after init the storage_class in every config.